index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE>
+<html>
+<head>
+     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Trial project</title>
+<link rel="stylesheet" href="styles.css"
```

Comment:
please, keep your codebase indentation to make your code more readable. The same type of codes, need to be on the same line. Hint: line 4 up to 7 needs to be parallel.